source/common/json/json_streamer.h

```diff
@@ -38,6 +38,26 @@ namespace Json {
 #define ASSERT_LEVELS_EMPTY ASSERT(this->levels_.empty())
 #endif
 
+class Constants {
+public:
+  // Constants for common JSON values.
+  static constexpr absl::string_view True = "true";
+  static constexpr absl::string_view False = "false";
+  static constexpr absl::string_view Null = "null";
+};
+
+// Simple abstraction that provide a output buffer for streaming JSON output.
```

Comment:
Add note that this is in anticipation of changing to a template.

Otherwise it's kind of pointless.

Reply:
Haha, of course. Just take it easy. I know what I am doing. :) That PR will coming soon after we landed this.